src/main/scala/com/databricks/spark/redshift/RedshiftRelation.scala

```diff
@@ -96,12 +108,12 @@ case class RedshiftRelation(jdbcWrapper: JDBCWrapper, params: MergedParameters,
   }
 
   protected def compileValue(value: Any): Any = value match {
-    case stringValue: String => s"\\'${escapeSql(stringValue.toString)}\\'"
+    case stringValue: UTF8String => s"\\'${escapeSql(stringValue.toString)}\\'"
```

Comment:
Why are we using `UTF8String` here, given that it's not intended to be a public API?